noteorganiser/frames.py

```diff
@@ -837,6 +844,23 @@ def resetSize(self):
         self.font.setPointSize(self.defaultFontSize)
         self.text.setFont(self.font)
 
+    def setupAutoRefresh(self, source):
+        """add current file to QFileSystemWatcher and refresh when needed"""
+        # only setup if wanted
+        if self.info.refreshEditor:
+            self.fileSystemWatcher.addPath(source)
+            self.fileSystemWatcher.fileChanged.connect(
+                self.getAutoRefreshSignal)
+            self.log.info("added file %s to FileSystemWatcher" % source)
+
+    def getAutoRefreshSignal(self):
```

Comment:
so `setupRefresh` and `refresh` ?

Doesn't that get confusing with the refresh methods in `Shelves` and `Library` that do something else? (allways redraw the UI instead of only refresh if the notebook is unmodified)